torch/csrc/jit/interpreter.cpp

```diff
@@ -478,6 +478,13 @@ struct InterpreterStateImpl {
       outputs.clear();
       loadTensorsFromRegisters(stage.outputs, outputs);
   }
+  const TensorType & tensorTypeForInput(size_t i) const {
```

Comment:
This is the only method (besides runOneStage) which depends on the `current_stage`; it might be worth pointing out that it is the input *for the current stage*, either in the method name or a comment.